docs/administration/service_management.md

```diff
@@ -1,132 +1,170 @@
-
 ## Service Listing Overview
 
-Let's look at the columns and data in the service table:
+The service listing table provides comprehensive information on each of the services configured in Assemblyline. Below is a detailed explanation of each column and the type of data it represents:
 
 ![Service Management page](./images/service_mgmt_page.png)
 
-The "Name" column contains the service name, and the "Version" column contains the version of the service that is running on Assemblyline. 
+**Name**
+
+The **Name** column contains the name of the service. This is the identifier that you will use to refer to this service throughout the system.
+
+**Version**
+
+The **Version** column lists the current running version of the service.
+
+**Category**
+
+The **Category** column categorizes the services into predefined groups. This helps in organizing services and understanding the role each service plays. Available categories out-of-the-box include:
+
+- **Antivirus**: Services that scan files for malware.
+- **Dynamic Analysis**: Services that execute the file in a sandbox environment to observe behaviors.
+- **External**: Services that rely on external data sources or submit data to external systems.
+- **Extraction**: Services that extract files from archives or other compound file formats.
+- **Filtering**: Services that eliminate certain files from being processed further.
+- **Internet Connected**: Services that need to connect to the internet to fetch updates or check data.
+- **Networking**: Services that analyze network traffic or related data.
+- **Static Analysis**: Services that analyze files without executing them.
+
+**Stage**
+
+The **Stage** column delineates at which point in the analysis pipeline a service will execute. The stages determine the sequence in which services operate, ensuring an organized and efficient workflow. Each service belongs to a single stage, executed in the following order:
+
+- **FILTER**: Initial filtering of files (e.g., Safelist). This stage determines whether subsequent stages will analyze a file.
+- **EXTRACT**: File extraction and unpacking (e.g., Extract, Unpacker). Services in this stage focus on decomposing files from archives or other compound formats.
+- **CORE**: Core analysis tasks. This stage is where the majority of services perform their analysis.
+- **SECONDARY**: Additional analysis performed with context from previous stages (e.g., VirusTotal). These services offer further insights based on the initial core analysis.
+- **POST**: Subsequent processing steps (e.g., URLCreator). Services in this stage act upon the results generated by earlier stages.
+- **REVIEW**: This final stage involves services that review the entire context of the analysis before performing their own tasks (e.g., Ancestry).
+
+**File Types**
+
+The **File Types** column specifies the Assemblyline file types that the service will accept for processing. This is defined using regular expressions. For instance:
```

Comment:
`Assemblyline file type` is a "thing" in AL and I wonder if there is a documentation page for it, that could list the possible values (or some possible values)?  Or are those following some kind of international standard?

Reply:
I'm not sure this is something we can do. We have a ton of file types and technically, any instance of Assemblyline can add as many as they want, so long as they have services that support those file types. What I would suggest, is that users may go to the "Administration" tab and select "File Identification", which will show what files the system currently is able to identify. https://malware-pb.cyber.gc.ca/admin/identify

Reply:
It would just be a constantly evolving list, so having one documentation page dedicated to it might be a constantly moving target

Reply:
I don't have access to the admin section, but I understand your comment.  Is it a free form field, or there is some kind of structure?  Are they all like `category/name`, or some could simply be like `name`?  

In short, you need to know what you are looking for...

Reply:
Assemblyline uses multiple format to identify file types, including: Libmagic, Trusted Mimes, Magic/Regex Patterns, and Yara Rules.